plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -314,7 +314,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 
 				rbacv1helpers.NewRule(Write...).Groups(autoscalingGroup).Resources("horizontalpodautoscalers").RuleOrDie(),
 
-				rbacv1helpers.NewRule(Write...).Groups(batchGroup).Resources("jobs", "cronjobs").RuleOrDie(),
+				rbacv1helpers.NewRule(Write...).Groups(batchGroup).Resources("jobs", "cronjobs", "cronjobs/finalizers").RuleOrDie(),
```

Comment:
I'm not sure this is the correct approach. I'm inclined to drop the `blockOwnerDeletion` setting from create job, but I'll let folks from sig-auth to speak about it.